### PR TITLE
AST-1430 - added  e2e test case for off canvas design settings

### PR DESCRIPTION
**Pull request by bsfsnehal**
### Description

e2e test case for off canvas background color, close icon color and popup padding
### How has this been tested?

npm run test:e2e:interactive — tests/e2e/specs/customizer/header-builder/mobile-header/off-canvas-header/offcanvas-design.test.js

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/header-builder/mobile-header/off-canvas-header/offcanvas-design.test.js [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+		await page.waitForSelector( '.ast-mobile-popup-drawer.active .menu-toggle-close' );
+		await expect( {
+			selector: '.ast-mobile-popup-drawer.active .menu-toggle-close',
+			property: 'color',
+		} ).cssValueToBe( `${ offCanvasColor[ 'off-canvas-close-color' ] }` );
+	} );
+
+	//popup padding
+	it( 'padding should apply correctly for tablet', async () => {
+		const fullscreenPopupPadding = {
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/header-builder/mobile-header/off-canvas-header/offcanvas-design.test.js:41]: Need camelcase variable declaration like'fullScreenPopupPadding'

Comment (bsfsnehal) [reply tests/e2e/specs/customizer/header-builder/mobile-header/off-canvas-header/offcanvas-design.test.js]: resolved

File: tests/e2e/specs/customizer/header-builder/mobile-header/off-canvas-header/offcanvas-design.test.js [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+			},
+			'off-canvas-close-color': 'rgb(21, 21, 21)',
+		};
+		await setCustomize( offCanvasColor );
+
+		await createNewPost( {
+			postType: 'page',
+			title: 'QA',
+		} );
+		await publishPost();
```
Comment (SimranChougule1) [inline tests/e2e/specs/customizer/header-builder/mobile-header/off-canvas-header/offcanvas-design.test.js:19]: need to update publishPost()

File: tests/e2e/specs/customizer/header-builder/mobile-header/off-canvas-header/offcanvas-design.test.js [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+
+		await page.goto( createURL( '/' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await setBrowserViewport( 'medium' );
+		await page.click( '.main-header-menu-toggle' );
+		await page.waitForSelector( '.ast-mobile-popup-drawer.active .ast-mobile-popup-inner' );
```
[the remark this answers is not in the document]
Comment (bsfsnehal) [reply tests/e2e/specs/customizer/header-builder/mobile-header/off-canvas-header/offcanvas-design.test.js:28]: The BG color is white and close icon is black, I have changed the colors.

**Pull request closed without merging**